hack/make-rules/test-cmd.sh

```diff
@@ -2324,6 +2324,26 @@ __EOF__
   kube::test::get_object_assert "nodes 127.0.0.1" "{{.spec.unschedulable}}" '<no value>'
 
 
+
+  ########################
+  # authorization.k8s.io #
+  ########################
+
+  # check remote authorization endpoint, kubectl doesn't actually display the returned object so this isn't super useful
+  # but it proves that works
```

Comment:
I don't know that we want a ton of output on kubectl create by default.  It
will be extremely difficult to take a random list of objects and make a
usable, effective output.  We'd probably want to err on a very succinct
result, and I doubt our ability to do that generically.

On Wed, Aug 3, 2016 at 3:55 PM, David Eads <notifications@github.com> wrote:

> In hack/make-rules/test-cmd.sh
> <https://github.com/kubernetes/kubernetes/pull/20573#discussion_r73408443>
> :
>
> > @@ -2324,6 +2324,26 @@ __EOF__
> >    kube::test::get_object_assert "nodes 127.0.0.1" "{{.spec.unschedulable}}" '<no value>'
> >
> >
> > +
> > +  ########################
> > +  # authorization.k8s.io #
> > +  ########################
> > +
> > +  # check remote authorization endpoint, kubectl doesn't actually display the returned object so this isn't super useful
> > +  # but it proves that works
>
> Just thinking aloud here: If we had called these request and response
> instead of spec and status, then we could have had a convention in kubectl
> that if a thing has request and response, then kubectl should print out the
> response.
>
> I think we probably want a "show me what came back" flag on kubectl create
> so that users can see how their object was defaulted.
>
> @fabianofranz <https://github.com/fabianofranz> Want to try to make such
> a thing?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/20573/files/e6849f08dfa738a8416da170c4566c92d23199d3#r73408443>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABG_p33msBM-8vog6dI3gHu5NybjmpWFks5qcPI2gaJpZM4HSwZB>
> .
>